backend/internal/service/oidc_service.go

```diff
@@ -277,6 +277,67 @@ func (s *OidcService) createTokenFromRefreshToken(refreshToken, clientID, client
 	return accessToken, newRefreshToken, 3600, nil
 }
 
+func (s *OidcService) IntrospectToken(clientID, clientSecret, tokenString string) (introspectDto dto.OidcIntrospectionResponseDto, err error) {
+	if clientID == "" || clientSecret == "" {
```

Comment:
RFC 7662 says that the introspection endpoint should also support refresh tokens too?

Reply:
True. Actually even a good usecase for this endpoint, since there is no client-side ability to verify the validity of a refresh token.

Reply:
Done in 324981dc.